packages/frontend/src/hooks/useQueryError.ts

```diff
@@ -23,11 +23,11 @@ const useQueryError = (): Dispatch<SetStateAction<ApiError | undefined>> => {
                     // drawer
```

Comment:
we don't have a drawer anymore, right? will be good if we search for that keyword and remove any unnecessary mentions of it